trace.go

```diff
@@ -139,11 +145,18 @@ func GetFields(err error) map[string]interface{} {
 
 // WrapWithMessage wraps the original error into Error and adds user message if any
 func WrapWithMessage(err error, message interface{}, args ...interface{}) Error {
+	return wrapWithMessage(1, err, message, args...)
```

Comment:
While it is sort of redundant (in that Wrap implements this use-case as well even simply by delegating to it), we cannot remove it as it might be used in a project.